app/views/email/PasscodeErrorView.scala.html

```diff
@@ -0,0 +1,36 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+@import views.html.templates.inputs.Text
+
+@this(mainTemplate: MainTemplate, text: Text, form: FormWithCSRF)
+
+@()(implicit user: User[_], messages: Messages, appConfig: AppConfig)
+
+@mainTemplate(title = messages("passcode.attempts.title")) {
+
+<h1 class="heading-large">@messages("passcode.attempts.title")</h1>
+
+    <p>@messages("passcode.attempts.information")@messages("common.fullstop")</p>
+
+    <p>
+    <a href="@appConfig.vatOverviewUrl">@messages("passcode.attempts.vatAccount")</a>
+        @messages("passcode.attempts.retry")@messages("common.fullstop")
```

Comment:
You could add the full stop to the last message itself to avoid using a separate full stop message here.

The full stop message is typically only used for when there's a full stop needed after a link, as link text should not contain a full stop.